invenio_requests/assets/semantic-ui/js/invenio_requests/request/RequestMetadata.js

```diff
@@ -33,15 +33,24 @@ const Community = ({ community }) => (
     <a href={`/communities/${community.slug}`}>{community.metadata.title}</a>
   </div>
 );
+const Email = ({ email }) => (
+  <div className="flex">
+    <Icon name="mail" className="mr-5" />
+    <span>{i18next.t("External email")}: {email.id}</span>
+  </div>
+);
 
-const UserOrCommunity = ({ userData, details }) => {
+const EntityDetails = ({ userData, details }) => {
   const isUser = "user" in userData;
   const isCommunity = "community" in userData;
+  const isEmail = "email" in userData;
```

Comment:
Is it always external when `email`? If yes, very minor:
```suggestion
  const isExternalEmail = "email" in userData;
```
and same for methods/components.

Reply:
Yes, indeed – since there's no notion of "internal" emails (unless they are associated with a user account), they are necessarily "external".
Fixed!